lib/std/build/InstallRawStep.zig

```diff
@@ -298,7 +334,26 @@ fn containsValidAddressRange(segments: []*BinaryElfSegment) bool {
     return true;
 }
 
-fn emitRaw(allocator: Allocator, elf_path: []const u8, raw_path: []const u8, format: RawFormat) !void {
+pub const Options = struct {
+    format: ?RawFormat = null,
+    only_section_name: ?[]const u8 = null,
```

Comment:
Can you allow filtering for more than a single section? Maybe just allow that field to be a slice that will filter out sections when set to non-zero length

Reply:
Sure. Do you think a callback `includeSection(name: []const u8) bool` would be even better, or just the list?

Reply:
Also, I'm not sure dumping multiple sections through this is a good idea. The small difference between this and dumping all of them is that this doesn't seek to the section base address before writing to the file.